WooCommerce/Classes/Tools/Connectivity/DefaultConnectivityObserver.swift

```diff
@@ -27,9 +32,9 @@ final class DefaultConnectivityObserver: ConnectivityObserver {
     func updateListener(_ listener: @escaping (ConnectivityStatus) -> Void) {
```

Comment:
I think this function is not needed anymore, now that we have a shared publisher, right?